plugins/out_s3/s3.c

```diff
@@ -1083,13 +1101,25 @@ static int s3_put_object(struct flb_s3 *ctx, const char *tag, time_t create_time
         final_body = body;
         final_body_size = body_size;
     }
+
+    char final_body_md5[25];
```

Comment:
fluent bit C code style role: always declare variables only at the top of functions. This is noted here: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md